quantiles.json

```diff
@@ -21,20 +21,29 @@
         },
         {
             "name": "probabilities",
-            "description": "A list of probabilities to calculate quantiles for. The probabilities must be between 0 and 1 (inclusive).",
-            "schema": {
-                "type": "array",
-                "items": {
-                    "type": "number",
-                    "minimum": 0,
-                    "maximum": 1
+            "description": "Quantiles to calculate. Either a list of probabilities or the number of intervals:\n\n* Provide an array with a list of probabilities to calculate quantiles for. The probabilities must be between 0 and 1 (inclusive).\n* Provide an integer to specify the number of intervals to calculate quantiles for. Calculates q-quantiles with equal-sized intervals.",
```

Comment:
Something to discuss or add I think: what should happen if the list of probabilities is not sorted (e.g. user specifies `[0.5, 0.1, 0.9]`?
- throw error
- back-end automatically sorts probabilites and returns 10%, 50% and 90% quantiles in that order
- back-end returns quantiles in order 50%, 10%, 90%

Reply:
What does make most sense? I guess it's not the error?

Reply:
Silently sorting the probabilities back-end side is indeed the worst option.

The error option is probably the simplest to start with (in terms of documenting and implementing it), and it can be relaxed later if there is interest in that.

However: `numpy.quantile` apparently supports returning the quantiles in the same order as the probabilities:
 ```
>>> np.quantile(range(100), [0.5, 0.1, 0.9])
array([49.5,  9.9, 89.1])
```
and I guess it's  fairy straightforward to support that anyway in other implementations

Reply:
For simplicity and interoperability, I'm happy to go with the error if no one else objects. We can then still open it up later if desired.

Reply:
Changed, please review.